<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModManagerWidget</class>
 <widget class="QWidget" name="ModManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MessageWidget" name="gameRunningMessage" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btnRefresh">
        <property name="toolTip">
         <string>Refreshes the mod list. Does NOT reload all mods.</string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnInstallMod">
        <property name="toolTip">
         <string>Adds a SilVerPLuM mod from a *.zip file. For non-SilVerPLuM mods use the mod importer in the arrow menu.</string>
        </property>
        <property name="text">
         <string>Add mod</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDownloadMod">
        <property name="text">
         <string>Mod repository</string>
        </property>
        <property name="icon">
         <iconset theme="applications-internet">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="CategoryComboBox" name="categoryFilter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBar">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Search ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MessageWidget" name="message" native="true"/>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="ModManagerEntryList" name="modList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>783</width>
        <height>334</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="DependencyCheckerWidget" name="dependencyWarning" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionReloadMods">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reload all mods</string>
   </property>
   <property name="toolTip">
    <string>Reloads all mods from the mod directory.</string>
   </property>
  </action>
  <action name="actionImportMod">
   <property name="icon">
    <iconset theme="document-import">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import from files</string>
   </property>
   <property name="toolTip">
    <string>Imports a mod from mods that are not compatible with SilVerPLuM</string>
   </property>
  </action>
  <action name="actionAddDefaultMods">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add default mods</string>
   </property>
   <property name="toolTip">
    <string>Adds the mods coming by default with SilVerPLuM</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QWidget</extends>
   <header>messagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DependencyCheckerWidget</class>
   <extends>QWidget</extends>
   <header>dependencycheckerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CategoryComboBox</class>
   <extends>QComboBox</extends>
   <header>categorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ModManagerEntryList</class>
   <extends>QWidget</extends>
   <header>modmanagerentrylist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
